tools/github/github-utils.ts

```diff
@@ -53,16 +52,18 @@ export async function getCommitAndLabels(
 
   const prByLabels = {};
   for (const commit of commits) {
-    const prs = await octokit.rest.repos.listPullRequestsAssociatedWithCommit({
-      owner,
-      repo,
-      commit_sha: commit.sha,
-    });
-    for (const pr of prs.data) {
-      if (pr.labels && pr.labels.length > 0) {
-        for (const label of pr.labels) {
+    const foundPrsNumbers = commit.commit.message.split("\n")[0].match(/\(#([0-9]+)\)/);
```

Comment:
This will include the reverts too ?
like `Revert "First step in implementing #4386 (#4437)`

Reply:
Good catch, we need to exclude the revert case